node/hack/repair_solana/repair.go

```diff
@@ -172,8 +183,8 @@ func main() {
 							continue
 						}
 						if nseq != p.Sequence {
-							log.Printf("%d != %d, offset +%d", nseq, p.Sequence, nseq-p.Sequence)
-							offset += nseq - p.Sequence
+							offset += 1
```

Comment:
what does this do?

Reply:
It advances the offset by one if we landed on the wrong seq no - it's just a dumb brute force way of narrowing down the set of txs to request (for lack of an index we could query)